docs/sources/source/tts/tutorial.rst

```diff
@@ -114,38 +121,6 @@ torch.distributed.launch module and sepcifying the num_gpus as the
 
 Inference
 ---------
-You can now to inference with either your own trained Tacotron 2, or you can
-use our pre-trained Tacotron 2 model. Please download our models for
-`tacotron 2 here <https://ngc.nvidia.com/catalog/models/nvidia:tacotron2_ljspeech>`_ and 
-`waveglow here <https://ngc.nvidia.com/catalog/models/nvidia:waveglow_ljspeech>`_.
-Next create the texts that you want
-to generate and add them to a json like the training dataset. They should
-have lines like so:
-
-.. code-block:: json
-
-    {"audio_filepath": "", "duration": 1.0, "text": "Talk to me!"}
-    {"audio_filepath": "", "duration": 1.0, "text": "Speech Synthensis is cool."}
-
-For Mandarin, they should have lines like so:
-
-.. code-block:: json
-
-    {"audio_filepath": "", "duration": 1.0, "text": "jin1 tian1 tian1 qi4 bu2 cuo4."}
-    {"audio_filepath": "", "duration": 1.0, "text": "ni3 kan4 bao4 zhi3 ma0"}
-
-The "text" should contain the **pinyin** sequence in Mandarin. The digit behind each Chinese character's pinyin is the **tone**. 0 stands for soft tone.
-
-Inference can be done with the tts_infer.py file under the
-NeMo/examples/tts folder like so:
-
-.. code-block:: bash
-
-    python tts_infer.py --spec_model=tacotron2 --spec_model_config=configs/tacotron2.yaml --spec_model_load_dir=<directory_with_tacotron2_checkopints> --vocoder=waveglow --vocoder_model_config=configs/waveglow.yaml --vocoder_model_load_dir=<directory_with_waveglow_checkopints> --save_dir=<where_you_want_to_save_wav_files> --eval_dataset <mainfest_to_generate>
-
-For Mandarin, remember to replace the config file of Tacotron 2 with tacotron2_mandarin.yaml.
-
-.. tip::
-    The inference via waveglow can be further controlled via the ``--waveglow_denoiser_strength`` and
-    ``--waveglow_sigma`` arguments. If there is a stuble white noise to the audio, we recommend slowing increasing
-    ``--waveglow_denoiser_strength`` from 0.
+For inference in English, please refer to our
```

Comment:
Maybe we should keep the above information regarding - --waveglow_sigma and --waveglow_denoiser_strength (or at the least, include a small comment regarding these hyper parameters in the notebook itself).